packages/visx-demo/src/sandboxes/visx-delaunay-voronoi/Example.tsx

```diff
@@ -37,6 +37,8 @@ function Example({ width, height, margin = defaultMargin }: VoronoiProps) {
   const innerWidth = width - margin.left - margin.right;
   const innerHeight = height - margin.top - margin.bottom;
 
+  if (innerWidth < 0 || innerHeight < 0) return null;
```

Comment:
an early return statement will break the rules of hooks, where they would then be called conditionally/a different # of times per render.

maybe a different approach (since we already return `null` if `width < 10`), is to add something like `const innerWidth = Math.max(0, width - margin.left - margin.right)`

Reply:
I completely forgot about the rules of hooks. 😅
I've pushed a commit reflecting your comment [here](https://github.com/airbnb/visx/pull/1758/commits/fb47cc74a860556ffdc4ad808ee6b63fae3eebeb). Thank you!